goldens/public-api/compiler-cli/compiler_options.d.ts

```diff
@@ -7,6 +7,7 @@ export interface I18nOptions {
     enableI18nLegacyMessageIdFormat?: boolean;
     i18nInLocale?: string;
     i18nUseExternalIds?: boolean;
+    i18nNormalizeExternalTemplateLineEndingsInICUs?: boolean;
```

Comment:
my attempt at making this shorter and based on existing naming precedences... what do you think?
```suggestion
    i18nNormalizeExternalTemplateEOLsInICUs?: boolean;
```

I don't love this, because it's still too cryptic and long, also consider `i18nNormalizeEOLsInICUs` - yes, it's more ambiguous but we can clarify in the docs that this applies only to external templates (the definition of "external" will need to be part of the docs anyway, as most people don't understand what "external template" means).

Reply:
My suggestion is `i18nNormalizeLineEndingsInICUs`.
It is not 100% accurate because line endings in ICUs are always normalized for inline templates - and this flag only affects the normalization of ICU expressions in external templates. But I think that it is close enough and short enough.


Igor replied:

> I like your suggestion, but can we also take into consideration that tsconfig.json has a somewhat related precedence for this: https://www.typescriptlang.org/v2/en/tsconfig#newLine we might want to take that into account when deciding the name and intentionally decide to either follow it if the concepts are the same, or intentionally decide to diverge if we are trying to disambiguate between two similar concepts.

Reply:
This is actually not really related that that standard tsconfig setting. That is only about outputting line endings. Whatever the value of that setting, TS always normalizes line endings in backticked strings.

Reply:
great thanks for the context in that case `i18nNormalizeLineEndingsInICUs` sgtm

Reply:
please re-request the review when the changes are made.